src/popup.html

```diff
@@ -222,6 +230,51 @@ <h2 class="fx-relay-panel-header-title i18n-content" data-i18n-message-id="popup
           </div>
         </settings-panel>
 
+         <!-- Panel: CSAT Survey -->
+         <survey-panel data-panel-id="survey" class="is-hidden fx-relay-panel" id="survey-panel">
+          <header class="fx-relay-panel-header">
+            <!-- Back Button -->
+            <button data-nav-level="root" data-back-target="masks" class="fx-relay-panel-header-btn-back">
+              <img class="i18n-alt-tag" data-i18n-message-id="returnToPreviousPanel" src="/icons/nebula-back-arrow.svg" alt="">
+            </button>
+            
+            <!-- Panel Title -->
+            <h2 class="fx-relay-panel-header-title i18n-content" data-i18n-message-id="popupSurveyLink" />
+
+          </header>
+          
+          <div class="fx-relay-panel-content">
+             <!-- Satisfaction Selection Form -->
+            <div class="csat-survey-content">
+              <div class="csat-survey-section">
+                <p class="csat-survey-text i18n-content" data-i18n-attribute="title" data-i18n-message-id="popupCsatSurveyText" /> 
+              </div>
+
+              <div class="csat-survey-satisfaction-section">
+                <!-- Select issue checkboxes -->
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="0" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionVeryDissatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="1" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionDissatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="2" data-event-action="click"  data-i18n-message-id="popupCsatSurveySatisfactionNeutral"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="3" data-event-action="click"  data-i18n-message-id="popupCsatSurveySatisfactionSatisfied"></button>
+                <button class="i18n-content fx-relay-csat-button" type="button" data-satisfaction="4" data-event-action="click" data-i18n-message-id="popupCsatSurveySatisfactionVerySatisfied"></button> 
```

Comment:
These elements are not currently keyboard accessible.

Reply:
I believe the buttons are already quite accessible by virtue of being actual button elements, which are natively focusable and accessible to screen readers. I did add an aria-label though.